assets/src/components/nav.tsx

```diff
@@ -15,19 +17,30 @@ const Nav: React.FC<Props> = ({ children }) => {
   const [, dispatch] = useContext(StateDispatchContext)
   const deviceType = useScreenSize()
 
+  const {
+    currentView: { openView, selectedVehicleOrGhost },
+  } = usePanelStateFromStateDispatchContext()
+  const isViewOpen =
+    openView !== OpenView.None || (selectedVehicleOrGhost && true) || false
+
```

Comment:
suggestion(non-blocking): what do you think about computing this in `usePanelStateForViewState`?

Reply:
Oh I like that idea! Will make the change.

Reply:
26a627da

Reply:
I like it!
I do have a question: Do you think that it is confusing that `isViewOpen` is not under `currentView`?
I prefer that `currentView` contains only the (reducer) state of the page, but that makes me question if we should be exposing `curentView` as it's own object vs it's properties being added to the "base object".

I am fine either way, just wondering your opinion.

Reply:
Hmm, good question, though I think for the reason you stated I prefer not adding `isViewOpen` to `currentView`. Maybe hoisting the properties of `currentView` to the main object would make sense at some point but I don't think that it's worth doing right now.